src/Components/Admins/Form/index.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState, useEffect } from 'react';
+import styles from './form.module.css';
+
+const Form = (props) => {
+  const [formData, setFormData] = useState({
+    firstName: '',
+    lastName: '',
+    dni: '',
+    phone: '',
+    email: '',
+    city: '',
+    password: ''
+  });
+
+  const [isEditing, setIsEditing] = useState(false);
+
+  useEffect(() => {
+    if (props.selectedAdmin) {
```

Comment:
I think this value is always null since you never set a value for `selectedAdmin` and that might be the reason why the data does not preload when you want to edit.

Reply:
Changed it! Now selectedAdmin has a value.